src/main/java/List.java

```diff
@@ -0,0 +1,194 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class List {
```

Comment:
Can consider a more specific class name, List is also the name of a built-in java interface, maybe something like "TaskList"